python/lsst/faro/measurement/MatchedCatalogMeasurementTasks.py

```diff
@@ -288,7 +288,7 @@ class AMxTask(Task):
     ConfigClass = AMxConfig
     _DefaultName = "AMxTask"
 
-    def run(self, matchedCatalog, metric_name):
+    def run(self, metric_name, matchedCatalog):
```

Comment:
No documentation (which needs to be fixed everywhere) but also can we have some consistency in variable names? Other `run` methods use `metricName` instead but also mixing `metric_name` with `matchedCatalog` is jarring. Pick one and stick with it (underscores are preferred these days)

Reply:
I recall from discussion 13 April 2021 that we decided on camelCase convention for faro package.

Reply:
I'd rather have name_metric and catalog_matched but since most of the rest of the package is camelCase I'll switch it to metricName.